src/pages/Share/SubmitTab.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {saveUnknownUserDetails} from '@libs/actions/Share';
+import Navigation from '@libs/Navigation/Navigation';
+import MoneyRequestParticipantsSelector from '@pages/iou/request/MoneyRequestParticipantsSelector';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+function SubmitTab() {
+    return (
+        <MoneyRequestParticipantsSelector
+            iouType={CONST.IOU.TYPE.SUBMIT}
+            onFinish={() => {}}
+            onParticipantsAdded={(value) => {
+                const participant = value.at(0);
+                const reportID = participant?.reportID;
+                const accountID = participant?.accountID;
+                if (accountID && !reportID && participant) {
```

Comment:
```suggestion
                if (accountID && !reportID) {
```

In case `accountID` is truthy, `participant` will also be truthy. herefore, no need to include `participant` in the statement.